### PR TITLE
add a button for resetting all filters

### LINKED ISSUE #129: Add refresh button
Related to #88.
> - It may be helpful to include a 'reset selections' button to help users reset their filter choices so they can smoothly start a new query. Alternatively you could consider a clear all selections button. This would be useful as currently users need to unclick each drug if they only want to see one or two in the plot.

### PR DESCRIPTION
**Pull request by billwan96**
closes #129

### CODE REVIEW

File: src/modules/components/sidebar.py [hunk 1]
```diff
@@ -63,6 +63,9 @@
             )
         ]),
         dbc.Row([html.Hr()]),
+        dbc.Row([html.A(
+            dbc.Button("Reset", color="secondary", className="me-1"),
+            href='/')]),
```
Comment (SoloSynth1) [inline src/modules/components/sidebar.py:68]:
```suggestion
        dbc.Row([html.A(
            dbc.Button(
                "Reset All Filters",color="light",
                className="me-1 link-underline-opacity-0", outline=True),
            href='/', className="d-grid gap-2 link-underline-opacity-0")]),
```

**Review by SoloSynth1 — COMMENTED** [review]
@billwan96 Thanks for the PR! Please see the suggestion in the comment.

File: src/modules/components/sidebar.py (hunk 1 above)
Comment (billwan96) [reply src/modules/components/sidebar.py]: Thank you for the suggestion! @SoloSynth1 I have tried and committed them.

**Review by SoloSynth1 — APPROVED** [review]
LGTM, thanks @billwan96! Merging into `main`.

**Pull request merged**